setup.py

```diff
@@ -29,8 +29,10 @@
             'PyMySQL >= 0.6.2',
             'sqlparse >= 0.1.16',
             'configobj >= 5.0.6',
-            'pycrypto >= 2.6.1',
             ],
+        extras_require={
+            'parse_mylogin_cnf': ['pycrypto >= 2.6.1']
+        },
```

Comment:
This makes it so pycrypto is never installed by default. It is only installed when someone goes out and does `pip install pycrypto`. 

I'd prefer to take the approach of not installing pycrypto in Windows but install it everywhere else. 

See https://github.com/dbcli/pgcli/pull/397 for reference on how to do this.